src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width: 96px;
+  height: 16px;
```

Comment:
why do we need to define width and height? just remove `space-between` style and add `margin-right` to each stars exclude the last one